scipy/stats/_continuous_distns.py

```diff
@@ -2924,13 +2924,13 @@ def fit(self, data, *args, **kwds):
 
 
 class gamma_gompertz_gen(rv_continuous):
-    """ A Gamma/Gompertz continuous random variable
+    u""" A Gamma/Gompertz continuous random variable
```

Comment:
Python 2 support is dropped hence this is redundant now.

Reply:
Thanks for pointing it out. I have been facing difficulty in resolving the check error, so I tried that. Under expectation, it did not work :(